config/yurtadm/cluster-image/README.md

```diff
@@ -0,0 +1,13 @@
+# Build an OpenYurt ClusterImage
+
+`yurtadm init` is implemented by sealer, you can modify the kubefile to make your own openyurt cluster image.
+
+```bash
+cd openyurt-latest
+
+# build openyurt ClusterImage
+sealer build -t registry-1.docker.io/your_dockerhub_username/openyurt-cluster:latest-k8s-1198 -f Kubefile .
```

Comment:
Maybe it is better to explain how to install sealer and the version of sealer.

Reply:
ok.